media/css/m24/components/navigation-refresh.scss

```diff
@@ -114,17 +114,8 @@ $margin-top: 54px; // top margin offset for mobile navigation menu
     margin: 4px 0 0;
     padding: 0;
     position: relative;
-
-    &::after {
-        background: $color-black;
-        bottom: 4px;
-        content: "";
-        display: block;
-        height: 1px;
-        left: 0;
-        position: absolute;
-        width: 100%;
-    }
+    text-decoration: underline;
+    text-underline-offset: 0.2em;
```

Comment:
I think the fix height is to vertically center the menu button to the logo as well as the fix height for open nav menu on mobile. It might be worth opening a separate issue to tackle

Reply:
I think there are some nav updates coming anyway. So I'm going to consider the nav large font issues resolved for now.